docker-build/history.txt

```diff
@@ -0,0 +1,80 @@
+    1  cd /media/beherith/VBox_GAs_7.0.6/
+    2  ls
+    3  sudo ./autorun.sh 
+    4  ./autorun.sh 
+    5  sudo su
+    6  su root
+    7  sudo ls
+    8  sudo nano /etc/default/locale
```

Comment:
delete the file (not because it's chaff but because it reveals you use `nano` as the editor. we can't take humiliating public PR hits like that)